platforms/testplatform/templates/global-announcement/PUSH.mustache

```diff
@@ -1,2 +1,8 @@
-title: "{{ message.title }}"
-description: "{{ language.description }}"
+notification:
+    title: "{{ message.title }}"
+    body: "{{ language.description }}"
+apns:
```

Comment:
I was wrong, we can actually remove the ´apns´ part here. Firebase uses the `notification` part to create notifications for all the different platforms, `apns` etc.  can be used for some platform-specific overrides.